DispatchR/Dispatchee.cs

```diff
@@ -116,5 +117,23 @@ internal bool ShouldExecute()
 
             return true;
         }
+
+        /// <summary>
+        /// Compares one instance to another
+        /// </summary>
+        /// <param name="other">The other dispatchee</param>
+        /// <returns>-1 if this comes before other, 1 if this comes after other, 0 otherwise</returns>
+        public int CompareTo(Dispatchee? other)
```

Comment:
Good call. Would like to follow same syntax as the result of the library to keep things consistent `if (condition) return;`